### PR TITLE
Added Winter events

### LINKED ISSUE #424: add Winter 2024 events to event page

### PR DESCRIPTION
**Pull request by jamesmwu**
# Changes

Updated event descriptions.
Fixes #424 

## Type of change

Please delete options that are not relevant.

- [X] Content Update (non-breaking change which updates the content of the website)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by katelynsyu — CHANGES_REQUESTED** [review]
You also need to add event dates in src/data/events.js, like when the indiv stackschool workshops are, etc

**Comment by jamesmwu** [comment]
Note:
Still missing assets for HOTH, location for HOTH TBA.

Abbreviated / shortened some of the Stackschool workshop names to prevent text overlapping the Details button of the card component-- I'm assuming it's intentional that the card component dimensions are hard-coded so height doesn't increase.

File: src/data/events.js [hunk 1]
```diff
@@ -103,6 +103,69 @@ const events = [
...
+	{
+		name: 'Stackschool #6: User Auth (Part 2)',
+		date: getDateTime(2024, 2, 22, 18),
+		location: 'Kaplan Hall 135',
+		imgFilePath: 'event/2024w-stackschool-banner.png',
+		detailLink: 'https://discord.gg/rup2p6fxA5'
+	},
+	{
+		name: 'Stackschool #7: External APIs',
+		date: getDateTime(2024, 3, 7, 18),
```
Comment (katelynsyu) [inline src/data/events.js:151]: Stackschool #7 is in Week 8 which should be 2/29

File: src/data/events.js [hunk 2]
```diff
@@ -103,6 +103,69 @@ const events = [
...
+		location: 'Kaplan Hall 135',
+		imgFilePath: 'event/2024w-stackschool-banner.png',
+		detailLink: 'https://discord.gg/rup2p6fxA5'
+	},
+	{
+		name: 'Stackschool #8: Project Showcase',
+		date: getDateTime(2024, 3, 14, 18),
```
Comment (katelynsyu) [inline src/data/events.js:158]: Should be 3/7/24

File: src/data/events.js [hunk 3]
```diff
@@ -103,6 +103,69 @@ const events = [
...
+		location: 'Kaplan Hall 135',
+		imgFilePath: 'event/2024w-stackschool-banner.png',
+		detailLink: 'https://discord.gg/rup2p6fxA5'
+	},
+	{
+		name: 'Hack On The Hill XII',
+		date: getDateTime(2024, 3, 2, 8),
+		location: 'Location to be announced!',
+		imgFilePath: 'event/hack-banner.png',
+		detailLink: 'https://discord.gg/rup2p6fxA5'
```
Comment (katelynsyu) [inline src/data/events.js:168]: detailLink should be hoth website [hoth.uclaacm.com](https://hoth.uclaacm.com)

File: src/data/events.js [hunk 4]
```diff
@@ -103,6 +103,69 @@ const events = [
...
+	},
+	{
+		name: 'Stackschool #8: Project Showcase',
+		date: getDateTime(2024, 3, 14, 18),
+		location: 'Kaplan Hall 135',
+		imgFilePath: 'event/2024w-stackschool-banner.png',
+		detailLink: 'https://discord.gg/rup2p6fxA5'
+	},
+	{
+		name: 'Hack On The Hill XII',
```
Comment (katelynsyu) [inline src/data/events.js:164]: We're on HOTH XI not XII

File: src/data/events.js [hunk 5]
```diff
@@ -103,6 +103,69 @@ const events = [
 		location: 'Kaplan Hall A65',
 		imgFilePath: 'event/2023f-hackschool-banner.png',
 		detailLink: 'https://discord.gg/rup2p6fxA5'
+	},
+	{
```
Comment (katelynsyu) [inline src/data/events.js:107]: Can we get rid of all of last quarter's events pls?

**Review by katelynsyu — CHANGES_REQUESTED** [review]

**Comment by jamesmwu** [comment]
This is what I get for using copilot

**Review by katelynsyu — APPROVED** [review]
lgtm. boo copilot

**Pull request merged**